dev-python/panel/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>sci@gentoo.org</email>
+		<name>Gentoo Science Project</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
ALLARCHES? BTW, are you interested in maintaining any of these (too?) - if so, add yourself above sci

Reply:
That's... a good question. My volunteer time is fairly scarce, but that's true for everyone here. Rather than officially signing up as proxy maintainer, I'd kinda prefer to occasionally push PRs bumping ebuilds at my leisure.

With apologies, would you mind if we left this as is for the moment?